README.md

```diff
@@ -411,6 +411,25 @@ change the UUID.
 You can't. This is because historical bookings still refer to that prison.
 Disable it instead (see above).
 
+#### Merging two prisons
+
+If two formerly-separate parts of an estate now have the same booking team and
```

Comment:
I'm open to any suggestions. I think it's explained in the next section, though.